challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |string| string.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.find_all { |i|  i.downcase == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.select { |i|  i.downcase != "dog"  }
```

Comment:
You could also `reject` the ones that do match "dog". I'm not sure one is better than the other.